From 0fc9bee85ab994043e1c5365d665fe51c95cb047 Mon Sep 17 00:00:00 2001
From: Nick Richards <nick@endlessm.com>
Date: Tue, 31 Jul 2018 18:09:09 +0100
Subject: [PATCH] appdata: Reword description

Use the appdata description to list the available features and try to make the copy more inviting
---
 data/org.gnome.Music.appdata.xml.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/data/org.gnome.Music.appdata.xml.in.in b/data/org.gnome.Music.appdata.xml.in.in
index 999236e0..6e8295c5 100644
--- a/data/org.gnome.Music.appdata.xml.in
+++ b/data/org.gnome.Music.appdata.xml.in
@@ -7,7 +7,10 @@
   <summary>Play and organize your music collection</summary>
   <description>
     <p>
-      Music is the new GNOME music playing application.
+      An easy way to play your music. Automatically discover music on your computer, the local network and internet services.
+    </p>
+    <p>
+      Find tracks in your local collection, get music from DLNA servers or try something new with the Jamendo and Magnatune services.
     </p>
   </description>
   <screenshots>
-- 
2.17.0

